src/lib/ci.js

```diff
@@ -0,0 +1,47 @@
+
+export const CI_HOST = "https://lighthouse-dot-webdotdevsite.appspot.com";
+
+export async function fetchMedians() {
+  const resp = await window.fetch(`${CI_HOST}/lh/medians?url=all`);
+  return await resp.json();
+}
+
+export async function runLighthouse(url, signedIn=false) {
+  const body = {
+    url,
```

Comment:
Can you comment these params? I'm not sure I grok what `replace` is doing?

Reply:
I'm not sure either—it was from the old codebase. I've left you a TODO :)